compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/types/BLangArrayType.java

```diff
@@ -62,10 +65,14 @@ public String toString() {
         final StringBuilder[] sb = {new StringBuilder(getTypeName())};
         if (sizes.length == 0) {
             Arrays.stream(sizes).forEach(size -> {
-                if (size == -1) {
-                    sb[0].append("[]");
+                if (size instanceof BLangLiteral) {
```

Comment:
Let's use node kind.

Reply:
Is there a specific reason for this? Because if I don't specifically check for the `BLangLiteral` type in the `if` statement, my current logic won't stand as the else block assumes that it is receiving `BLangLiteral` Type.